### PR TITLE
chore: rename PlayerStatus to Player

### PR DESCRIPTION
**Pull request by AjimenezDCL**
Small naming refactor to make a following PR smaller.

`PlayerStatus` has been renamed to `Player`

### CODE REVIEW

**Review by sandrade-dcl — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarNamesHUD/AvatarNamesHUDView.cs [hunk 1]
```diff
@@ -78,9 +78,9 @@ public void TrackPlayer(PlayerStatus playerStatus)
             }
 
             AvatarNamesTracker tracker = reserveTrackers.Dequeue();
-            tracker.SetPlayerStatus(playerStatus);
+            tracker.SetPlayerStatus(player);
```
Comment (Suduck) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarNamesHUD/AvatarNamesHUDView.cs:81]: This should be `SetPlayer `now?

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/UsersAroundListHUD/IUsersAroundListHUDListView.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ public interface IUsersAroundListHUDListView
     event Action<bool> OnRequestMuteGlobal;
     event Action OnGoToCrowdPressed;
     event Action OnOpen;
-    void AddOrUpdateUser(PlayerStatus playerStatus);
+    void AddOrUpdateUser(Player player);
```
Comment (Suduck) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/UsersAroundListHUD/IUsersAroundListHUDListView.cs:9]: This should be `AddOrUpdatePlayer`?

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/UsersAroundListHUD/UsersAroundListHUDListView.cs [hunk 3]
```diff
@@ -55,14 +55,14 @@ void OnDestroy()
         Settings.i.OnGeneralSettingsChanged -= OnSettingsChanged;
     }
 
-    void IUsersAroundListHUDListView.AddOrUpdateUser(PlayerStatus playerStatus)
+    void IUsersAroundListHUDListView.AddOrUpdateUser(Player player)
```
Comment (Suduck) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/UsersAroundListHUD/UsersAroundListHUDListView.cs:58]: Same as before

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs [hunk 4]
```diff
@@ -309,17 +309,17 @@ private void MapRenderer_OnSceneInfoUpdated(MinimapMetadata.MinimapSceneInfo sce
             scenesOfInterestMarkers.Add(sceneInfo, go);
         }
 
-        private void OnOtherPlayersStatusAdded(string userId, PlayerStatus playerStatus)
+        private void OnOtherPlayersStatusAdded(string userId, Player player)
```
Comment (Suduck) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs:312]: This should be `OnOtherPlayersAdded`?

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs [hunk 5]
```diff
@@ -309,17 +309,17 @@ private void MapRenderer_OnSceneInfoUpdated(MinimapMetadata.MinimapSceneInfo sce
...
-            marker.gameObject.name = $"UserIcon-{playerStatus.name}";
+            marker.gameObject.name = $"UserIcon-{player.name}";
             marker.gameObject.transform.SetParent(overlayContainer.transform, true);
-            marker.Populate(playerStatus);
+            marker.Populate(player);
             usersInfoMarkers.Add(userId, poolable);
         }
 
-        private void OnOtherPlayerStatusRemoved(string userId, PlayerStatus playerStatus)
+        private void OnOtherPlayerStatusRemoved(string userId, Player player)
```
Comment (Suduck) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs:322]: Same as before

**Review by Suduck — APPROVED** [review]
LGTM! Just a few method rename suggestions that you decide to implement or not!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs (hunk 5 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs:322]: Yeah, I trusted a bit too much on Rider auto-renaming xD. Thanks, nice catch

**Review by BrianAmadori — APPROVED** [review]

**Pull request merged**